components/Accordion/Accordion.vue

```diff
@@ -39,16 +43,19 @@ import type { AccordionState } from '~/types/form'
 import { key, type AccordionRegister } from '~/components/Accordion/injectionKey'
 
 const props = withDefaults(defineProps<{
-  id: string | undefined
+  id?: string | undefined
```

Comment:
Why remove the optional here?

Reply:
I'm adding it, because it was not optional until now, it was only allowing an undefined value

Reply:
Read it the other way :-)